packages/asc-ui/src/components/Header/examples/DataportaalHeader.stories.tsx

```diff
@@ -78,7 +91,52 @@ const DataportaalHeader: React.FC<{}> = () => (
   />
 )
 
-storiesOf('Composed/Header', module).add(
-  'implementation for dataportaal',
-  () => <DataportaalHeader />,
-)
+export const DataportaalBody: React.FC<PageLayoutInterface> = ({
+  tall,
+  fullWidth,
+  children,
+}) => <Body {...{ tall, fullWidth }}>{children}</Body>
+
+storiesOf('Composed/Header', module)
+  .addDecorator(storyFn => (
+    <div
+      style={{
+        backgroundColor: 'lightgrey',
+        height: '100%',
+        position: 'relative',
+      }}
+    >
+      {storyFn()}
+    </div>
+  ))
+  .add('implementation for dataportaal - Frontpage', () => (
+    <>
+      <DataportaalHeader tall fullWidth={false} />
+      <DataportaalBody tall fullWidth={false}>
+        <BodyFiller />
+      </DataportaalBody>
+    </>
+  ))
+  .add('implementation for dataportaal - Editorial content', () => (
```

Comment:
I think one example is enough... we just want to show an example of how we implement the header in a project.

Reply:
Not really agree, the different states of a header (even though atlas specific) should be pictured in the examples, is the best way to document this.